Remotion/Web/Core/ContentSecurityPolicy/CspDirectives.cs

```diff
@@ -0,0 +1,176 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+namespace Remotion.Web.ContentSecurityPolicy;
+
+//
+// !!! NOTE: Changes also need to be done in `CspHeader.GetCspDirectiveName`.
```

Comment:
Instead of the comment, please add an test that iterates over all items in the CspEnum and tries to make an entry.  Then we can simply add a a failure message that tells us what to do.

Reply:
Edit: I saw EnumValuesNamesTest so I think we alreayd have this test, correct. plz drop the note.